main/accounts/index.ts

```diff
@@ -105,7 +106,8 @@ export class Accounts extends EventEmitter {
       log.info(`Account ${address} not found, creating account`)
 
       const created = 'new:' + Date.now()
-
+      const accountMetaId = uuidv5(address, accountNS)
+      const { name = '' } = store('main.accountsMeta', accountMetaId) || {}
```

Comment:
did you test this change? when I add a new account now it has no label, guessing it's because we're overwriting whatever name is passed in with a blank string here

Reply:
No, I didn't test creating new accounts.  Yet another case for enabling linting:

https://eslint.org/docs/latest/rules/no-shadow

Reply:
why was this resolved? this is still not working. also the code is still shadowing a now-unused variable

Reply:
whoops my bad, was looking at an outdated file :( all good!